### PR TITLE
Upgrade Swift SDK to use grpc-swift-1.0.0-alpha.10

### PR DESCRIPTION
**Pull request by kapitoshka438**

### CODE REVIEW

File: docs-src/content/using-cubic-sdk/installation.md [hunk 1]
```diff
@@ -46,6 +46,6 @@ Once you have your Swift package set up, adding swift-cubic as a dependency is a
 
 ```swift
 dependencies: [
-    .package(url: "https://github.com/cobaltspeech/sdk-cubic.git", .upToNextMajor(from: "1.4.0"))
+    .package(url: "https://github.com/cobaltspeech/sdk-cubic.git", .upToNextMajor(from: "1.4.1"))
```
Comment (happyalu) [inline docs-src/content/using-cubic-sdk/installation.md]: version change should be a separate PR.

File: grpc/swift-cubic/Cubic.swift [hunk 2]
```diff
@@ -12,4 +12,4 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-public let CubicVersion = "1.4.0"
+public let CubicVersion = "1.4.1"
```
Comment (happyalu) [inline grpc/swift-cubic/Cubic.swift]: version change should be a separate PR.

File: docs/using-cubic-sdk/installation/index.html [hunk 3]
```diff
@@ -261,7 +261,7 @@ <h4 id="swift-package-manager">Swift Package Manager</h4>
 
 <p>Once you have your Swift package set up, adding swift-cubic as a dependency is as easy as adding it to the <code>dependencies</code> value of your <code>Package.swift</code>.</p>
 <div class="highlight"><pre style="color:#d0d0d0;background-color:#202020;-moz-tab-size:4;-o-tab-size:4;tab-size:4"><code class="language-swift" data-lang="swift">dependencies: [
-    .package(url: <span style="color:#ed9d13">&#34;https://github.com/cobaltspeech/sdk-cubic.git&#34;</span>, .upToNextMajor(from: <span style="color:#ed9d13">&#34;1.4.0&#34;</span>))
+    .package(url: <span style="color:#ed9d13">&#34;https://github.com/cobaltspeech/sdk-cubic.git&#34;</span>, .upToNextMajor(from: <span style="color:#ed9d13">&#34;1.4.1&#34;</span>))
```
Comment (happyalu) [inline docs/using-cubic-sdk/installation/index.html]: docs need to be regenerated (refer old version)

File: Makefile [hunk 4]
```diff
@@ -49,14 +50,17 @@ ${DEPSVENV}/.done:
...
-		rm -f nio.tar.gz
-
-	cd ${DEPSSWIFT}/bin && make plugins 
+		"https://github.com/grpc/grpc-swift/archive/${SWIFT_GRPC_VERSION}.tar.gz" && \
+ 		tar xzf ${SWIFT_GRPC_VERSION}.tar.gz && \
+ 		mv grpc-swift-${SWIFT_GRPC_VERSION} bin && \
+ 		rm -f ${SWIFT_GRPC_VERSION}.tar.gz
+ 		cd ${DEPSSWIFT}/bin && make plugins
+ 		cp protoc-gen-grpc-swift ${BINPATH}
+ 		cp protoc-gen-swift ${BINPATH}
```
Comment (happyalu) [inline Makefile]: Where is BINPATH defined? Should this be DEPSBIN?

Comment (kapitoshka438) [reply Makefile]: Oops

**Comment by kapitoshka438** [comment]
> I'm happy to approve once Jenkins passes.

It passed. Please be happy.
